### PR TITLE
Support kubecfg

### LINKED ISSUE #15: Enable users to check their id-token github teams
The idea is to simplify this process:

https://user-guide.cloud-platform.service.justice.gov.uk/documentation/other-topics/check-id-token.html#checking-the-github-groups-in-your-id-token

Something like:

```
cloud-platform kubecfg show-github-teams
```

This would inspect your ~/.kube/config file (or optionally take a filename as an argument), internally decode the id-token JWT and output just the github teams section, as per the user guide article.

### PR DESCRIPTION
**Pull request by mogaal**
This PR closes #15 , it adds functionality to parse your default kubeconfig file (`~/.kube/config`) and print your tokens claims. It also has a flag to specify a custom kubeconfig file with `-f`

### CODE REVIEW

File: pkg/commands/kubecfg.go [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	Short: `Cloud Platform kubeconfig related commands`,
+}
+
+var kubecfgShowGithubTeamsCmd = &cobra.Command{
+	Use:   "show-github-teams",
+	Short: `Checking the GitHub groups in your ID token`,
+	Example: heredoc.Doc(`
+	$ cloud-platform kubecfg show-github-teams
+	`),
+	RunE: func(cmd *cobra.Command, args []string) error {
```
Comment (digitalronin) [inline pkg/commands/kubecfg.go:33]: Could you add a PreRun hook here to call the self-upgrade method?

File: pkg/kubecfg/fixtures/kubeconfig.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+current-context: ""
+kind: Config
+preferences: {}
+users:
+- name: alejandro.garrido@digital.justice.gov.uk
+  user:
+    auth-provider:
+      config:
+        client-id: m1yfMHQDYSxncbhmUNIrm1yfMHQDYSxn
+        client-secret: 50fk9Tlc1ESIfUE4CKSCEUo3mYCKfw7mNVKmOcg1fMAnQEMFyLaKF2Kie8XG
```
Comment (digitalronin) [inline pkg/kubecfg/fixtures/kubeconfig.yml:21]: Am I right in thinking that these are real-looking but not actually real client-secret and id-token values?

File: pkg/kubecfg/githubTeams_test.go [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+func eq(t *testing.T, got interface{}, expected interface{}) {
+	t.Helper()
+	if !reflect.DeepEqual(got, expected) {
+		t.Errorf("expected: %v, got: %v", expected, got)
+	}
+}
+
+func TestGetTokenClaims(t *testing.T) {
+	token := "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkFsZWphbmRybyBHYXJyaWRvIE1vdGEiLCJlbWFpbCI6ImFsZWphbmRyby5nYXJyaWRvQGRpZ2l0YWwuanVzdGljZS5nb3YudWsiLCJodHRwczovL2s4cy5pbnRlZ3JhdGlvbi5kc2QuaW8vZ3JvdXBzIjpbbnVsbCwiZ2l0aHViOndlYm9wcyIsbnVsbF0sIm5pY2tuYW1lIjoibW9nYWFsIiwiaWF0IjoxNTE2MjM5MDIyfQ._5Vkfh1lDWQvV9xIcbpnMFWOnkJBzCcp1X7lYcEG9sE"
```
Comment (digitalronin) [inline pkg/kubecfg/githubTeams_test.go:16]: As per my question above, these are real-looking but not actual secrets, right?

File: pkg/commands/kubecfg.go [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+}
+
+var kubecfgCmd = &cobra.Command{
+	Use:   "kubecfg",
+	Short: `Cloud Platform kubeconfig related commands`,
+}
+
+var kubecfgShowGithubTeamsCmd = &cobra.Command{
+	Use:   "show-github-teams",
```
Comment (digitalronin) [inline pkg/commands/kubecfg.go:28]: I think "id-token" or "id-token-claims" might be a better name for this command.

Comment (mogaal) [reply pkg/commands/kubecfg.go]: That makes more sense, just did it.

File: pkg/kubecfg/githubTeams_test.go (hunk 3 above)
Comment (mogaal) [reply pkg/kubecfg/githubTeams_test.go:16]: Yes, this token was generated used https://jwt.io/ , you can write fields you want in the "Decoded" side and it builds the token for you.

File: pkg/kubecfg/fixtures/kubeconfig.yml (hunk 2 above)
Comment (mogaal) [reply pkg/kubecfg/fixtures/kubeconfig.yml:21]: They are not real, just generated a random string [here](http://www.unit-conversion.info/texttools/random-string-generator/)

File: pkg/commands/kubecfg.go [hunk 5]
```diff
@@ -30,6 +30,7 @@ var kubecfgShowGithubTeamsCmd = &cobra.Command{
 	Example: heredoc.Doc(`
 	$ cloud-platform kubecfg id-token-claims
 	`),
+  PreRun: upgradeIfNotLatest,
```
Comment (digitalronin) [inline pkg/commands/kubecfg.go]: I think you need to run `make fmt`

**Pull request merged**